### PR TITLE
CLOUDP-237136: [mongocli] fix packing

### PR DESCRIPTION
**Pull request by jeroenvervaeke**
## Proposed changes
Fix packing

_Jira ticket:_ CLOUDP-237136

### CODE REVIEW

File: tools/genevergreen/generate/generate.go [hunk 1]
```diff
@@ -181,7 +181,7 @@ func PostPkgMetaTasks(c *shrub.Configuration, toolName string) {
 func PublishStableTasks(c *shrub.Configuration, toolName string) {
 	dependency := []shrub.TaskDependency{
 		{
-			Name:    fmt.Sprintf("compile_%s", toolName),
+			Name:    "compile",
```
Comment (andreaangiolillo) [inline tools/genevergreen/generate/generate.go:184]: Why was toolName causing the issue? you still have the toolName as input for the function.

Comment (jeroenvervaeke) [reply tools/genevergreen/generate/generate.go:184]: The `compile_mongocli` task was removed. The `compile` task was left in the mongocli branch.

**Comment by fmenezes** [comment]
@jeroenvervaeke let's take this a step further and remove toolname from everywhere

**Comment by fmenezes** [comment]
let's remove it from `generate-msi.sh` and all remaining .go files

**Comment by jeroenvervaeke** [comment]
> @jeroenvervaeke let's take this a step further and remove toolname from everywhere

I planned to do that in a followup pr. I've created this one quickly to unblock the release.
But I can do it here too if you prefer.

**Comment by fmenezes** [comment]
> > @jeroenvervaeke let's take this a step further and remove toolname from everywhere
> 
> I planned to do that in a followup pr. I've created this one quickly to unblock the release. But I can do it here too if you prefer.

let's do it here, our release is Thursday we have time

File: tools/genevergreen/generate/generate.go [hunk 2]
```diff
@@ -207,57 +168,24 @@ func PublishStableTasks(c *shrub.Configuration, toolName string) {
 	}
 }
 
-func PublishSnapshotTasks(c *shrub.Configuration, toolName string) {
+func PublishSnapshotTasks(c *shrub.Configuration) {
 	dependency := []shrub.TaskDependency{
 		{
-			Name:    fmt.Sprintf("compile_%s", toolName),
+			Name:    "compile_mongocli",
```
Comment (fmenezes) [inline tools/genevergreen/generate/generate.go:174]:
```suggestion
			Name:    "compile",
```

File: tools/genevergreen/generate/generate_test.go [hunk 3]
```diff
@@ -24,18 +24,18 @@ import (
...
-		PublishSnapshotTasks(c, mongocli)
+		PublishSnapshotTasks(c)
 		assert.Len(t, c.Tasks, 34)
 		assert.Len(t, c.Variants, 2)
 	})
 }
 
 func TestPublishStableTasks(t *testing.T) {
-	t.Run(mongocli, func(t *testing.T) {
+	t.Run("mongocli", func(t *testing.T) {
```
Comment (fmenezes) [inline tools/genevergreen/generate/generate_test.go:36]: we don't need `t.Run` anymore, it is used when we have multiple test cases

Comment (jeroenvervaeke) [reply tools/genevergreen/generate/generate_test.go:36]: Updated

**Review by fmenezes — APPROVED** [review]

**Pull request merged**